recipe/meta.yaml

```diff
@@ -9,11 +9,12 @@ source:
   url: https://github.com/NVIDIA/nccl/archive/v{{ version }}.tar.gz
   sha256: 1923596984d85e310b5b6c52b2c72a1b93da57218f2bc5a5c7ac3d59297a3303
   patches:
-    # Upstreaming w/PR: https://github.com/NVIDIA/nccl/pull/854
-    - 0001-Allow-custom-NVCC-path.patch
+    # This isn't needed; setting CUDA_HOME sets NVCC correctly
+    #- 0001-Allow-custom-NVCC-path.patch
```

Comment:
Think one of the issues was handling the cross-compilation case. However if we are able to get cross-compilation working without it, maybe we should delete the patch and drop this comment

Reply:
ah, I see. thanks. Will check that

Reply:
your CI cross-compiles linux-aarch64 packages on a linux-64 build platform, IIRC?

Reply:
That's right

Reply:
I've got `CUDA_HOME = $BUILD_PREFIX` and `NVCC = $(CUDA_HOME)/bin/nvcc` as it would be for native compilation, I think that's right? (log [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=941710&view=logs&j=9a864fd9-6c8f-52ca-79ce-2aa6dca1a1de&t=10fc5aa2-324e-5982-4c88-6b31fcab16b3))

Reply:
(I don't have much experience with cross-compiling conda packages)

Reply:
AFAIU the difference from native is that the build prefix is populated with the target platform's compilers etc where appropriate, and the host prefix is populated with packages for the target platform?